tools/run_tests/jobset.py

```diff
@@ -177,6 +177,15 @@ def __cmp__(self, other):
     return self.identity() == other.identity()
 
 
+class JobResult(object):
+  def __init__(self):
+    self.state = "UNKNOWN"
+    self.returncode = -1
+    self.elapsed_time = 0
+    self.retries = 0
+    self.message = ""
```

Comment:
Then you'll need to check if None every time using it. You don't need to do this with empty string.